cmd/protoc-gen-goclay/genhandler/template.go

```diff
@@ -307,7 +305,12 @@ var (
 					panic(err)
 				}
 
-				ret = append(ret, aExpr.AssignableExpr(assignExpr)+" = &"+fMsg.GoType(goPkg)+"{}")
+				t := fMsg.GoType(goPkg)
+				// nullable option = 65001:0
+				if f.Target.FieldDescriptorProto.GetOptions().String() != "65001:0" {
```

Comment:
What do you think about moving this magic string to a constant?

Reply:
It is good idea. Done